### PR TITLE
Changed homepage content

### PR DESCRIPTION
**Pull request by Annedj**
Hey, I've just updated the home page, can you review it @lucien-george ?

image

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class PagesController <
+class PagesController < ApplicationController
```
Comment (luciengeorge) [inline app/controllers/pages_controller.rb:1]: Good catch 👌😅

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h1>Welcome!</h1>
+<p>We are doing amazing things!</p>
```
Comment (luciengeorge) [inline app/views/pages/home.html.erb]: let's add a little bit of margin on the sides 🙏

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (Annedj) [inline app/controllers/pages_controller.rb:1]: Good catch!

**Review by luciengeorge — COMMENTED** [review]
Thank you @Annedj 🙏 Just one little fix before we merge 👌

**Review by Annedj — COMMENTED** [review]
Good job!

**Comment by luciengeorge** [comment]
LGTM 👍 Merging this

**Pull request merged**